### ISSUE TITLE
'disabled'does not exist on type ControlButtonProps

### ISSUE THREAD

**Issue by Rubc** [wbkd/react-flow#2120]
### Build fails

When passing disabled to the ControlButton Component, we get this error

```
Type error: Type '{ children: Element; disabled: boolean; "data-testid": string; onClick: () => void; }' is not assignable to type 'IntrinsicAttributes & ControlButtonProps & { children?: ReactNode; }'.
  Property 'disabled' does not exist on type 'IntrinsicAttributes & ControlButtonProps & { children?: ReactNode; }'.

  321 |             <Icon color={6} icon={'settings'} size={'1rem'} />
  322 |           </ControlButton>
> 323 |           <ControlButton disabled={props.saving} data-testid='save-button' onClick={props.onSave}>
      |                          ^
  324 |             <Icon className={`${props.saving ? styles.disabled : ''}`} color={6} icon={'save'} size={'1rem'} />
  325 |           </ControlButton>
  326 |         </Controls>
error Command failed with exit code 1.
```

The disabled prop works as it's passed to the HTML button.

Can the type be updated to support this? 

### Steps to Reproduce the Bug or Issue

- Passing disabled to the ControlButton Component

- Executing: yarn build

### Expected behavior

The build to succeed

### Platform

- OS: Windows
- Browser: Chrome, Firefox
- Version: 10.0.0

**Comment by moklick** [comment]
You are right, good catch. We will add that!

**Comment by moklick** [comment]
Fix released in v10.2.1

**Comment by Rubc** [comment]
> Fix released in v10.2.1

@moklick  Thanks for solving it this quick!
